src/queries/genericItem.js

```diff
@@ -77,6 +77,10 @@ export const GET_GENERIC_ITEMS = gql`
       }
       dataProvider {
         id
+        logo {
+          id
```

Comment:
do we need the logo id in the queries that were updated?

Reply:
we previously had an issue with the apollo cache, where the cache would misbehave if one query included the id, while another did not.

to resolve that we decided to include all ids, so that we can not run into this issue again if we add another query that includes the id at a later point.

Reply:
forgot about that. thanks for pointing out.